docs/bitbucket-development.md

```diff
@@ -65,18 +65,18 @@ The Access and Refresh Tokens will be stored against the username and the userna
 
 # On-Premise Bitbucket
 
-On-premise Bitbucket, more correctly known as Bitbucket Server or Bitbucket DC, has a number of differences compared to the cloud instance of Bitbucket, https://bitbucket.org.
+On-premise Bitbucket, more correctly known as Bitbucket Server or Bitbucket DC, has a number of differences compared to the cloud instance of Bitbucket, [bitbucket.org](https://bitbucket.org).
```

Comment:
Should we have short names for links in this document similar to those added in `CODE_OF_CONDUCT.md`?

Reply:
Yes! 🙂 
I will work on doing this for all markdown files.

At the same time I will replace `aka.ms` links that refer to other docs within this repo with relative links, so as to:

1. Remove dependency on MS short link service
2. Allow links to work offline.

Please let me know if this latter part is not wanted.

Reply:
I am ok with the latter part, but I think @mjcheetham may want to weigh in on this.